cmd/kcp/kcp.go

```diff
@@ -97,6 +97,11 @@ func main() {
 				return errors.NewAggregate(errs)
 			}
 
+			// rnu as early as possible to avoid races later when some components (e.g. grpc) start early using klog
+			if err := completed.GenericControlPlane.Logs.ValidateAndApply(); err != nil {
+				return err
```

Comment:
i see we don't use the pattern of `return fmt.Errorf("error validating and applying generic control plane logs: %w, err)`. I am a bit worried this will not expose the full stack trace picture and logs and will make debugging harder? (not related to this PR as I see the same a few lines below).

Reply:
Validation errors are usually pretty good with context. Don't think we need this here.